src/api/app/models/maintenance_incident.rb

```diff
@@ -62,17 +62,17 @@ def project_name
     "#{maintenance_db_project.name}:#{incident_id}"
   end
 
-  def getUpdateinfoCounter(time, template = '%Y-%C')
+  def get_updateinfocounter(time, template = '%Y-%C')
```

Comment:
```suggestion
  def get_updateinfo_counter(time, template = '%Y-%C')
```
I would use capital letters as the clue where the underscores should have been

Reply:
Hm... you would use capital letters, and you suggest to use `get_updateinfo_counter`...

Reply:
I understand it the other way around: "I would use underscores as the clue where the capital letters should have been". On it...